src/app/components/CoinChart.tsx

```diff
@@ -122,9 +125,24 @@ export default function CoinChart({
               {coin?.id && "Volume 24h"}
             </span>
           ) : (
-            <span className="text-base leading-5 sm:text-xl font-normal dark:text-[#D1D1D1] text-[#191932] sm:leading-6 capitalize">
-              {coin?.id}
-            </span>
+            <>
+              <span className="text-base leading-5 sm:text-xl font-normal dark:text-[#D1D1D1] text-[#191932] sm:leading-6 capitalize">
+                {data.map((c) => {
+                  if (c.id === coin?.id) {
+                    return c.name;
+                  }
+                })}
```

Comment:
The map method is generally used when you want to transform each element in an array and create a new array of the same length. I don't think we are doing that here, we can use a forEach loop here.